cmake/templates/license.h.in

```diff
@@ -136,7 +136,36 @@ evmc:
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
+)"
+#cmakedefine EMSCRIPTEN
+#ifdef EMSCRIPTEN
```

Comment:
Not sure if we should instead just add it unconditionally.

Reply:
Also should we add a CC0 notice for the base64 snippet as well?

Reply:
I now changed it to unconditionally add the licenses (since e.g. evmc is also not part of solc itself) and added a notice following https://developer.mozilla.org/en-US/docs/MDN/About#copyrights_and_licenses (If I see it correctly via wayback machine, the snippets were added significantly after 2010 and thereby fall under the CC0 case).